packages/management/cliplugins/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: cliplugins.tanzu.vmware.com
+  namespace: tkg-system
```

Comment:
nit: this is just a placeholder, so this can be removed
I need to make an update to package scaffolding logic to not scaffold namespace key